### ISSUE TITLE
Main folder selector is blank

### ISSUE THREAD

**Issue by hockeymikey** [perfect-things/perfect-home#99]
I use the dropdown for the main folder, but it's blank, I don't have any options just a blank box of a selector. It does show the bookmarks of the bookmark menu though. I am using Waterfox classic, which may be a factor I am guessing.

**Comment by tborychowski** [comment]
Thanks for the feedback. Yeah, it is probably Waterfox (it runs on a dated engine, and has some differences from the regular firefox when it comes to browser APIs), but I'll have a look whenever I have some free time :-)

**Comment by tborychowski** [comment]
Unfortunately I'm not able to test this :-|
I've upgraded to MacOS Big Sur beta and Waterfox Classic does not work! (crashes).
I will retest in a couple of weeks, maybe they'll fix it :-)

**Comment by dpalacio** [comment]
I also see this, using Firefox-ESR 78.8.0. Is that version not supported?

**Comment by tborychowski** [comment]
Thanks for the update. I didn't even know that such a version exists :-)
I downloaded it and tested it and it works for me:
 
<img width="1073" alt="Screenshot 2021-03-13 at 10 17 10" src="https://user-images.githubusercontent.com/392513/111027122-1e5c1500-83e6-11eb-964a-0af4e98c69ba.png">

Did you create a new profile (it asked me when I first started it, so I created a new profile).

Lastly - I would be surprised if it didn't work, as the engine in v78 is the same as in v87 that I'm using and the APIs have not changed. 
I honestly can't figure out why it doesn't work for you... :-(
